ecommerce/views.py

```diff
@@ -226,3 +230,25 @@ def check_order_status(request, pk):
     order.save()
     
     return redirect('/ecommerce/order/all/')
+
+def product_search(request):
+    products = Product.objects.all()
+    category_object = Category.objects.all()
```

Comment:
Chú ý đặt cho đúng số nhiều số ít
```suggestion
    categories = Category.objects.all()
```